.env

```diff
@@ -1,5 +1,5 @@
 APP_ID_IOS=app.covidshield
-APP_ID_ANDROID=app.covidshield
+APP_ID_ANDROID=com.google.android.apps.exposurenotification
```

Comment:
Note: for easy testing, I figure we could use Google Sample Application Id here instead of CovidShield.

Reply:
Not sure we should commit this

Reply:
I would say yes but if we agree not to, I can add some docs to the README

Reply:
Ya, personally I think we should keep the APP_ID consistent and add a section for testing on Android with the example bundle id in README

Reply:
Added to README